### ISSUE TITLE
The DropDown control lost 'disabled' state on refresh.

### ISSUE THREAD

**Issue by Solwo** [OfficeDev/office-ui-fabric-react#419]
Scenario: The DropDown contol has a property 'disabled={false}'.
When a page loaded first time, the control disabled. Next time, when something changed on the page, the DropDown do re-render itself and becomes active. 

However, when I use old-style property  isDisabled={false},  the DropDown works as expected. 

It seems, the componentWillReceiveProps has a bug, it propagates only isDisabled property to next state?

```
constructor(props?: IDropdownProps) {
    super(props, {
      'isDisabled': 'disabled'
    });

    this.state = {
      id: getId('Dropdown'),
      isOpen: false,
      selectedIndex: this._getSelectedIndex(props.options, props.selectedKey),
      isDisabled: this.props.isDisabled || this.props.disabled
    };
  }

  public componentWillReceiveProps(newProps: IDropdownProps) {
    this.setState({
      selectedIndex: this._getSelectedIndex(newProps.options, newProps.selectedKey),
      isDisabled: newProps.isDisabled
    });
  }

```

**Comment by dzearing** [comment]
I investigated your issue and made a small patch. Will be published today. Thanks for reporting!

**Comment by dzearing** [comment]
https://github.com/OfficeDev/office-ui-fabric-react/pull/420

**Comment by dzearing** [comment]
Now published in 0.59.0. Thanks again.
